{
	"cve": {
		"data_type": "CVE",
		"data_format": "MITRE",
		"data_version": "4.0",
		"CVE_data_meta": {
			"ID": "CVE-2019-19926",
			"ASSIGNER": "cve@mitre.org"
		},
		"problemtype": {
			"problemtype_data": [
				{
					"description": [
						{
							"lang": "en",
							"value": "CWE-476"
						}
					]
				}
			]
		},
		"references": {
			"reference_data": [
				{
					"url": "http://lists.opensuse.org/opensuse-security-announce/2020-02/msg00010.html",
					"name": "openSUSE-SU-2020:0189",
					"refsource": "SUSE",
					"tags": []
				},
				{
					"url": "http://lists.opensuse.org/opensuse-security-announce/2020-02/msg00015.html",
					"name": "openSUSE-SU-2020:0210",
					"refsource": "SUSE",
					"tags": []
				},
				{
					"url": "http://lists.opensuse.org/opensuse-security-announce/2020-02/msg00025.html",
					"name": "openSUSE-SU-2020:0233",
					"refsource": "SUSE",
					"tags": []
				},
				{
					"url": "https://access.redhat.com/errata/RHSA-2020:0514",
					"name": "RHSA-2020:0514",
					"refsource": "REDHAT",
					"tags": []
				},
				{
					"url": "https://github.com/sqlite/sqlite/commit/8428b3b437569338a9d1e10c4cd8154acbe33089",
					"name": "https://github.com/sqlite/sqlite/commit/8428b3b437569338a9d1e10c4cd8154acbe33089",
					"refsource": "MISC",
					"tags": [
						"Patch",
						"Third Party Advisory"
					]
				},
				{
					"url": "https://security.netapp.com/advisory/ntap-20200114-0003/",
					"name": "https://security.netapp.com/advisory/ntap-20200114-0003/",
					"refsource": "CONFIRM",
					"tags": []
				},
				{
					"url": "https://usn.ubuntu.com/4298-1/",
					"name": "USN-4298-1",
					"refsource": "UBUNTU",
					"tags": []
				},
				{
					"url": "https://usn.ubuntu.com/4298-2/",
					"name": "USN-4298-2",
					"refsource": "UBUNTU",
					"tags": []
				},
				{
					"url": "https://www.debian.org/security/2020/dsa-4638",
					"name": "DSA-4638",
					"refsource": "DEBIAN",
					"tags": []
				},
				{
					"url": "https://www.oracle.com/security-alerts/cpuapr2020.html",
					"name": "N/A",
					"refsource": "N/A",
					"tags": []
				}
			]
		},
		"description": {
			"description_data": [
				{
					"lang": "en",
					"value": "multiSelect in select.c in SQLite 3.30.1 mishandles certain errors during parsing, as demonstrated by errors from sqlite3WindowRewrite() calls. NOTE: this vulnerability exists because of an incomplete fix for CVE-2019-19880."
				}
			]
		}
	},
	"configurations": {
		"CVE_data_version": "4.0",
		"nodes": [
			{
				"operator": "OR",
				"cpe_match": [
					{
						"vulnerable": true,
						"cpe23Uri": "cpe:2.3:a:sqlite:sqlite:3.30.1:*:*:*:*:*:*:*"
					}
				]
			}
		]
	},
	"impact": {
		"baseMetricV3": {
			"cvssV3": {
				"version": "3.1",
				"vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
				"attackVector": "NETWORK",
				"attackComplexity": "LOW",
				"privilegesRequired": "NONE",
				"userInteraction": "NONE",
				"scope": "UNCHANGED",
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"availabilityImpact": "HIGH",
				"baseScore": 7.5,
				"baseSeverity": "HIGH"
			},
			"exploitabilityScore": 3.9,
			"impactScore": 3.6
		},
		"baseMetricV2": {
			"cvssV2": {
				"version": "2.0",
				"vectorString": "AV:N/AC:L/Au:N/C:N/I:N/A:P",
				"accessVector": "NETWORK",
				"accessComplexity": "LOW",
				"authentication": "NONE",
				"confidentialityImpact": "NONE",
				"integrityImpact": "NONE",
				"availabilityImpact": "PARTIAL",
				"baseScore": 5.0
			},
			"severity": "MEDIUM",
			"exploitabilityScore": 10.0,
			"impactScore": 2.9,
			"acInsufInfo": false,
			"obtainAllPrivilege": false,
			"obtainUserPrivilege": false,
			"obtainOtherPrivilege": false,
			"userInteractionRequired": false
		}
	},
	"publishedDate": "2019-12-23T01:15Z",
	"lastModifiedDate": "2020-08-06T19:15Z"
}